.github/workflows/merge_and_release_to_staging.yml

```
@@ -37,11 +33,8 @@ jobs:
       - name: Release issues in Deriv.app
         uses: binary-com/fe-toolbox@production_V20230615_0
```

```suggestion
        uses: binary-com/fe-toolbox@production_V20230615_0
```

Will update version after [this](https://github.com/binary-com/fe-toolbox/pull/7) PR is merged